lib/python/qmk/cli/json2c.py

```diff
@@ -17,26 +18,26 @@ def json2c(cli):
 
     This command uses the `qmk.keymap` module to generate a keymap.c from a configurator export. The generated keymap is written to stdout, or to a file if -o is provided.
     """
-    # Error checking
+
+    # Parse the configurator from stdin
     if cli.args.filename and cli.args.filename.name == '-':
-        # TODO(skullydazed/anyone): Read file contents from STDIN
-        cli.log.error('Reading from STDIN is not (yet) supported.')
-        cli.print_usage()
-        return False
+        user_keymap = json.load(sys.stdin)
```

Comment:
I'd like to gracefully handle when a user provides an invalid json (and trust me, they will) instead of erroring out with an exception.

Reply:
Good idea. I have just added error handling, and I'll look into c2json command as well.